application/app/assets/stylesheets/connector.scss

```diff
@@ -0,0 +1,17 @@
+.badge-soft-danger {
+  background-color: rgba(220, 53, 69, 0.12); /* light red */
+  color: #842029;
```

Comment:
I wonder if we can create SCSS variables in a `vars.scss` file to define the colors and use the variables in our scss files, so that we can reuse values and be able to redefine the application theme. I would do it in another PR